src/gui/plugins/scene3d/Scene3D.cc

```diff
@@ -889,7 +889,7 @@ void IgnRenderer::HandleMouseEvent()
 /////////////////////////////////////////////////
 void IgnRenderer::BroadcastHoverPos()
 {
-  if (this->dataPtr->hoverDirty)
+  if (this->dataPtr->spawnPreview && this->dataPtr->hoverDirty)
```

Comment:
There are other plugins listening to the hover event, such as `TapeMeasure`:

https://github.com/ignitionrobotics/ign-gazebo/blob/24be610ac1e8889b6c538a110c84da7f91360a39/src/gui/plugins/tape_measure/TapeMeasure.cc#L235